cryptography/src/bls12381/primitives/group.rs

```diff
@@ -319,14 +364,14 @@ impl Element for G1 {
 }
 
 impl Point for G1 {
-    fn map(&mut self, data: &[u8]) {
+    fn map(&mut self, dst: DST, data: &[u8]) {
```

Comment:
coverage checker seems to be warning that this function isn't covered by any tests -- is this a concern?

Reply:
I'll add a G1 test 👍

Reply:
https://github.com/commonwarexyz/monorepo/pull/295/commits/1425c06b4c62699d776f3b4eb63fa994ccf727e5

There is potentially a follow-up here where we make the functions in `ops` support either approach (like `blst` does with `min_pk` and `min_sig`) but felt enforcing a single default could make things easier to understand.